mmv1/templates/terraform/examples/cloud_run_service_multiple_regions.tf

```diff
@@ -0,0 +1,230 @@
+# Cloud Run service replicated across multiple GCP regions
+
+######################################
+# Terraform (Project-wide resources) #
+######################################
+data "google_project" "app" {
+  project_id = var.project_id
+}
+
+resource "google_project_service" "compute_api" {
+  project                    = data.google_project.app.project_id
+  service                    = "compute.googleapis.com"
+  disable_dependent_services = true
+  disable_on_destroy         = false
+}
+
+resource "google_project_service" "run_api" {
+  project                    = data.google_project.app.project_id
+  service                    = "run.googleapis.com"
+  disable_dependent_services = true
+  disable_on_destroy         = false
+}
+
+# [START cloudloadbalancing_run_multiregion]
+############################
+# Input variables (config) #
```

Comment:
i think we can make all of these `vars` in the yaml above. i don't think we want them defined in the example as i think that will take up more space than we'd like.

Reply:
Done.